omero/analyze/index.html

```diff
@@ -63,7 +63,7 @@ <h4>OMERO.matlab toolbox</h4>
                 <h4>OMERO.mtools</h4>
                 <p>An optional suite of MATLAB-based image analysis tools.</p>
                 <a class="button hollow tiny" href="http://help.openmicroscopy.org/workflows-mtools.html" target="_blank">View user guide</a>
-                <a class="button hollow tiny" href="https://www.youtube.com/watch?v=ZliNETbTQLU&t=9s" target="_blank">View movie</a>
+                <a class="button hollow tiny" href="https://www.youtube.com/watch?v=ZliNETbTQLU&amp;t=9s" target="_blank">View movie</a>
```

Comment:
Interesting we don't start them quite at the beginning.

Reply:
I'd never noticed that before! I think I just copied them across from my browser...